src/components/input.tsx

```diff
@@ -0,0 +1,64 @@
+import { type VariantProps, cva } from 'class-variance-authority'
+import React, { forwardRef } from 'react'
+
+import { cn } from '@/utils/cn'
+
+const inputVariants = cva(
+  [
+    'w-full h-14 rounded-xl border border-2 px-4 text-base text-medium',
+    'focus:outline-none focus:border-focus',
+    'placeholder-disabled',
+  ],
+
+  {
+    variants: {
+      variant: {
+        default: 'border-base',
+      },
+      error: {
+        true: 'border-error',
+      },
```

Comment:
Instead of having a variant, we can just pass the error message and add the class accordingly.
```suggestion
```